### ISSUE TITLE
JSON import segfault and internal error

### ISSUE THREAD

**Issue by Nufflee** [duckdb/duckdb#6803]
### What happens?

DuckDB cannot import the [attached JSON files](https://github.com/duckdb/duckdb/files/11024915/files.zip) using `select * from read_json_auto('file.json', maximum_object_size=30000000);`. I tried manually specifying the `json_format` but that didn't help either.

Error:
```
Error: INTERNAL Error: Unexpected yyjson tag in ValTypeToString
```

### To Reproduce

Run `select * from read_json_auto('file.json', maximum_object_size=30000000);` on the attached files and observe a segfault or internal error respectively.

### OS:

Ubuntu 22.04, x64, WSL

### DuckDB Version:

v0.7.1 b00b93f0b1

### DuckDB Client:

CLI

### Full Name:

Vito

### Affiliation:

None

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Nufflee** [comment]
> Thanks for the report! This is likely a duplicate of #6768

Yup, seems similar, except that I also managed to get a segfault with a smaller file.

**Comment by lnkuiper** [comment]
Closed via #7478
